lib/compute/jenkins-main-node.ts

```diff
@@ -60,6 +77,8 @@ export class JenkinsMainNode {
 
   static readonly JENKINS_DEFAULT_ID_PASS_PATH: String = '/var/lib/jenkins/secrets/myIdPassDefault';
 
+  public readonly EFS_ID: string;
```

Comment:
Do we need a class level variable public variable for `EFS_ID`? Let's declare and use it locally.

Reply:
Its conditional statement so need it at class level

Reply:
We can declare it outside the condition and use it

Reply:
Since config element is static it throws an error. Making it private that will prevent outside access to this variable.
Thanks for catching this